tools/distrib/python/grpc_prefixed/generate.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2020 The gRPC Authors
```

Comment:
How do we hook this in to run on Kokoro?

Reply:
This generate script is one-off. It generates the source wheels as `v1.0.0`. After we upload them to PyPI, we won't need to execute this script again, unless we want to change grpc-prefixed package's behavior.

Reply:
Ah. Of course. :+1: